pyproject.toml

```diff
@@ -131,6 +131,7 @@ solr = ["sqlalchemy-solr >= 0.2.0"]
 elasticsearch = ["elasticsearch-dbapi>=0.2.9, <0.3.0"]
 exasol = ["sqlalchemy-exasol >= 2.4.0, <3.0"]
 excel = ["xlrd>=1.2.0, <1.3"]
+fastmcp = ["fastmcp>=2.8.1"]
```

Comment:
Noting that we can probably add `fastmcp` to `requirements/development.in` -> https://github.com/apache/superset/blob/master/requirements/development.in#L19

Once you add it there, you have to run this script to pin it in `requirements/development.txt` -> https://github.com/apache/superset/blob/master/scripts/uv-pip-compile.sh

Reply:
dine

Reply:
Is there any feature that we are planning to use from fastmcp that does not come with https://github.com/modelcontextprotocol/python-sdk out of the box? I think it's good to have it, just double checking if we are planning to use any of the additional features such as auth, clients, server proxying and composition, generating servers from REST APIs, dynamic tool rewriting, built-in testing tools, etc.?